src/Apps/Order/Routes/__tests__/Payment.test.tsx

```diff
@@ -216,9 +234,9 @@ describe("Payment", () => {
       onCompleted(creatingCreditCardSuccess)
     )
 
-    mount(<PaymentRoute {...testProps} />)
-      .find(ContinueButton)
-      .simulate("click")
+    const paymentRoute = mount(<PaymentRoute {...testProps} />)
+    fillAddressForm(paymentRoute, validAddress)
```

Comment:
It seems like some of the `fillAddressForm` calls could be replaced with `paymentRoute.find(Checkbox).props().onSelect(false)` or you don't even need to call it given that the shipping address is already there.